components/oauth-client/Cargo.toml

```diff
@@ -8,9 +8,9 @@ workspace = "../../"
 [dependencies]
 log = "*"
 reqwest = "*"
-serde = ">= 1.0.0"
-serde_derive = ">= 1.0.0"
-serde_json = ">= 1.0.0"
+serde = "*"
+serde_derive = "*"
+serde_json = "*"
 
```

Comment:
I still disagree with changing these constraints, as there will _never_ be a time that we want a version of serde that's `< 1.0`, but I'm not going to fight it.

Reply:
You're right, its mostly just a matter of keeping things consistent across the codebase where we are using "*" as the default unless there is a very specific reason (like for hyper) where we must have a special spec.